src/backend/TrafficCourts/Staff.Service/Services/DisputeService.cs

```diff
@@ -161,7 +161,7 @@ public async Task<Dispute> UpdateDisputeAsync(long disputeId, ClaimsPrincipal us
             updatedDispute.NoticeOfDisputeGuid,
             FileHistoryAuditLogEntryType.FRMK, // VTC staff has added a file remark for saving or updating a dispute in Ticket Validation
             GetUserName(user),
-            staffComment);
+            staffComment!);
```

Comment:
Yeah using `staffComment ?? string.Empty` makes more sense. `UpdateDisputeAsync` does not have a parameter for `staffComment` but `ToFileHistoryWithNoticeOfDisputeId` can be updated to accept null.

This is just a temporary change anyways. This will be mandatory back again when the front-end will be complete to capture staff comment.